news/2024/2024-07-28-the-perennial-concludes.md

```diff
@@ -0,0 +1,420 @@
+</div>
+
+## osu! originals
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+The Perennial stepped it up this year with six original songs: one for each round of the tournament, plus a second original in grand finals. Each of them had their own original artwork, and the majority had video introductions. Of course, the tournament’s final boss, the grand finals tiebreaker, had all that and a full storyboard. Check out all six songs, as with a variety of genres and styles, there’s sure to be something for everyone. If you are interested in the video intros and the maps for all these originals, those can be found on our [YouTube](https://youtube.com/c/ThePerennialYT).
+
+### Group Stage
+
+[![](https://assets.ppy.sh/beatmaps/2198890/covers/raw.jpg)](https://osu.ppy.sh/beatmapsets/2198890#osu/4653331)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/425/Songs/Doomsday_-_TGGT.mp3">
+    </audio>
+</div>
+
+### Round of 16 & Quarter Finals
+
+[![](https://assets.ppy.sh/beatmaps/2202795/covers/raw.jpg)](https://osu.ppy.sh/beatmapsets/2202795#osu/4662244)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-07-28-coriolis.mp3">
+    </audio>
+</div>
+
+### Semi Finals
+
+[![](https://assets.ppy.sh/beatmaps/2206504/covers/raw.jpg)](https://osu.ppy.sh/beatmapsets/2206504#osu/4672137)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-07-28-are-we-floating-in-space.mp3">
+    </audio>
+</div>
+
+### Finals
+
+[![](https://assets.ppy.sh/beatmaps/2209723/covers/raw.jpg)](https://osu.ppy.sh/beatmapsets/2209723#osu/4680237)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-07-28-chromatical-dysfunktion.mp3">
+    </audio>
+</div>
+
+### Grand Finals
+
+[![](https://assets.ppy.sh/beatmaps/2214194/covers/raw.jpg)](https://osu.ppy.sh/beatmapsets/2214194#osu/4691331)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-07-28-metal-globe.mp3">
+    </audio>
+</div>
+
+[![](https://assets.ppy.sh/beatmaps/2214238/covers/raw.jpg)](https://osu.ppy.sh/beatmapsets/2214238#osu/4691422)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-07-28-echoes-of-memoria.mp3">
+    </audio>
+</div>
+
+If you haven't seen the [video and storyboard](https://youtu.be/IjUqxihQKRI?si=-IQeBAT5wlf16nhZ) for *Echoes of Memoria* yet, you can check it out here:
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/IjUqxihQKRI?si=-IQeBAT5wlf16nhZ" frameborder="0" allowfullscreen></iframe>
+</div>
```

Comment:
you could just embed the video by itself 

![image](https://github.com/user-attachments/assets/eabfeac9-2d0f-4b1a-ac10-e8d3a8472392)